Bottom: e57a12c5f1165bedcf93507248e6b80cb77e4640
Top:    0331a59d7fbfa847d6500b4fcc82c799a69f15cc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:41:46 +0000

Turn off debug hacks in object generation


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index e3667b3..d60d25a 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1483,9 +1483,9 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   destroy_appearance (copper_appearance);
 
   /* DEBUG */
-  poly_M_Copy0 (&PCB->Data->outline, info.poly);
-  PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  poly_M_Copy0 (&PCB->Data->outline, info.poly);
+//  PCB->Data->outline_valid = true;
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
